container/README.md

```diff
@@ -6,29 +6,29 @@
 ## How to deploy
 
 Prerequisites:
-A postgres database with semver extension is needed. Using abrt/postgres-semver
+A postgres database with semver extension is needed. Using `quay.io/abrt/faf-db`
 image is recommended. Persistent storage is provided by means of a podman volume.
 A Redis container is required to run and schedule faf actions from the web UI (see below).
 Communication between ABRT Analytics, the database and Redis is ensured by running all three containers
 in the same podman pod.
 
-`podman volume create <volume_name>`  
-`podman pod create -p 5432:5432 -p 6379:6379 -p 8080:8080 --name <pod_name>`  
-`podman run --pod <pod_name> -v <volume_name>:/var/lib/pgsql/data -e POSTGRESQL_ADMIN_PASSWORD=scrt --name db -dit abrt/postgres-semver`
+`podman volume create faf-volume`
+`podman pod create -p 5432:5432 -p 6379:6379 -p 8080:8080 --name faf-pod`
+`podman run --pod faf-pod -v faf-volume:/var/lib/pgsql/data -e POSTGRESQL_ADMIN_PASSWORD=scrt --name db -dit quay.io/abrt/faf-db`
```

Comment:
Can you please convert all these blocks to the indented form?
```suggestion
    podman volume create faf-volume
    podman pod create -p 5432:5432 -p 6379:6379 -p 8080:8080 --name faf-pod
    podman run --pod faf-pod -v faf-volume:/var/lib/pgsql/data -e POSTGRESQL_ADMIN_PASSWORD=scrt --name db -dit quay.io/abrt/faf-db
```

Reply:
Good point -- updated.